src/urllib3/util/ssl_.py

```diff
@@ -320,13 +320,8 @@ def create_urllib3_context(
     # Enable post-handshake authentication for TLS 1.3, see GH #1634. PHA is
     # necessary for conditional client cert authentication with TLS 1.3.
     # The attribute is None for OpenSSL <= 1.1.0 or does not exist in older
```

Comment:
Can remove comment about older versions of Python.

Reply:
Thanks, I've removed that comment but added in a comment about SSLContext from pyOpenSSL.